src/cloud/docker/docker-containers-service.md

```diff
@@ -138,14 +158,30 @@ docker-compose exec varnish varnishadm ban req.url '~' '.'
 
 Container name |Docker base image | Ports exposed
 -------- | -------- | ---------------
-web | [magento/magento-cloud-docker-nginx](https://hub.docker.com/r/magento/magento-cloud-docker-nginx), which is based on the [centos](https://hub.docker.com/_/centos) Docker image | none
+web | [magento/magento-cloud-docker-nginx][nginx], which is based on the [centos] Docker image | none
 
-The Web container uses nginx to handle web requests after TLS and Varnish. This container passes all requests to the FPM container.
+The Web container uses nginx to handle web requests after TLS and Varnish. This container passes all requests to the FPM container to serve the PHP code. See [Request flow]({{site.baseurl}}/cloud/docker/docker-containers.html#request-flow).
 
-The nginx configuration for this container is the standard Magento [nginx config](https://github.com/magento-dockerhub/magento-cloud-docker/blob/master/images/nginx/1.9/etc/vhost.conf). This can be changed by mounting a new config using a volume.
+The nginx configuration for this container is the standard Magento [nginx config]. This can be changed by mounting a new config using a volume.
```

Comment:
```suggestion
The NGINX configuration for this container is the standard Magento [nginx config]. This can be changed by mounting a new config using a volume.
```